llvm/lib/TargetParser/Triple.cpp

```diff
@@ -1035,6 +1040,9 @@ static Triple::ObjectFormatType getDefaultFormat(const Triple &T) {
 
   case Triple::dxil:
     return Triple::DXContainer;
+
+  case Triple::native_cpu:
+    return Triple::UnknownObjectFormat;
```

Comment:
Why is this 'unknown'? Not a blocker. Just curious. Thanks

Reply:
Native CPU modules cannot be translated to any object file format directly, they will be adjusted to a concrete triple (such as `x86_64-unknown-linux-gnu`) and then processed in the normal way for that target. The object format will be whatever is normal for that concrete triple, typically ELF but it may be COFF on Windows. If any code attempts to determine the object format based on a `native_cpu` triple alone, there is not enough information to answer.